### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by danivicario**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
   <script type="text/javascript" src="js/bullet.js"></script>
   <script type="text/javascript" src="js/player.js"></script>
   <script type="text/javascript" src="js/obstacle.js"></script>
+  <script type="text/javascript" src="js/scoreboard.js"></script>
```
Comment (ta-web-mad) [inline index.html:12]: ey dani, this is wrong

File: js/scoreboard.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+var ScoreBoard = {
```
Comment (ta-web-mad) [inline js/scoreboard.js:2]: you forgot your comments to this obj literal

**Review by ta-web-mad — CHANGES_REQUESTED** [review]

File: js/scoreboard.js (hunk 2 above)
Comment (danivicario) [reply js/scoreboard.js:2]: you bitch

Comment (ta-web-mad) [reply js/scoreboard.js:2]: ey putilla, esto esta listo

**Review by ta-web-mad — APPROVED** [review]
ready to go!!!

**Pull request merged**